### DISCUSSION TITLE
fenced code exec

### DISCUSSION THREAD

**Discussion by basaran** [pngwn/MDsveX#365] [Q&A] [answered]
Hello,

Sorry, this is probably something simple but I can't find anywhere how to get the fenced js blocks to execute. I tried in the online version as well.

````
```js exec
let cat = "manuel";
```

{cat}
````

is there configuration or plugin needed?

**Comment by pngwn** [comment, marked as answer]
Hey. That syntax is no longer supported, you can just use `script` tags now.

```svelte
<script>
  let cat = "manuel";
</script>

{cat}

**Reply by basaran** [reply]
It's not the first time Google gave me old advice :) Thank you for your efforts, mdsvex is great. Would there be any plugins that does client side code execution? Thank you again.

**Reply by pngwn** [reply]
Any normal svelte code will execute on the client as expected. This would work for example:

```svelte
<script>
  import Comp from './Comp.svelte';

  let cat = "manuel";

  function handle_click() {
    console.log('click');
  }
</script>

# {cat}

<button on:click={handle_click}>HI</button>

<Comp />
```
